src/test/java/org/casbin/shiro/controller/rbac/PermitAnnotationController.java

```diff
@@ -12,32 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package org.casbin.shiro.controller;
+package org.casbin.shiro.controller.rbac;
 
-import org.casbin.shiro.authorize.rbac.annotation.PreAuth;
+import org.casbin.shiro.authorize.rbac.annotation.auth.EnforcerAuth;
+import org.casbin.shiro.authorize.rbac.annotation.permit.EnforcerPermit;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-public class RbacAuthAnnotationController {
+public class PermitAnnotationController {
```

Comment:
What does `Permit` here mean?

Reply:
I create a annotation name EnforcerPermit, This is the controller for test the annotation.

Reply:
OK